pkg/asset/machines/master.go

```diff
@@ -362,7 +363,17 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 
 		pool.Platform.Azure = &mpool
 
-		machines, err = azure.Machines(clusterID.InfraID, ic, &pool, string(*rhcosImage), "master", masterUserDataSecretName)
+		ssn, err := icazure.GetSession(installConfig.Azure.CloudName, installConfig.Azure.ARMEndpoint)
+		if err != nil {
+			return err
+		}
```

Comment:
Do we need a new session? It looks like there is already a session in this context